genref/config.yaml

```diff
@@ -25,6 +25,9 @@ markdownDisabled: false
 stripPrefix:
   - k8s.io/api/
   - k8s.io/apimachinery/pkg/apis/
+  - github.com/tengqm/kubeconfig/config/kubeadm/v1beta2.
```

Comment:
Hi @tengqm . Are these strings correct. I'm asking about the period at the end.

Reply:
The period was intentional. Thanks for asking.